tox.ini

```diff
@@ -3,9 +3,18 @@ minversion=2.0
 distshare={homedir}/.tox/distshare
 # make sure to update enviroment list on appveyor.yml
 envlist=
-     linting,py26,py27,py33,py34,py35,pypy,
-     {py27,py35}-{pexpect,xdist,trial},
-     py27-nobyte,doctesting,freeze,docs
+     linting
```

Comment:
You don't need commas when writing a multi-line option. `tox -l` shows all environments as expected:

```
$ tox -l
linting
py26
py27
py33
py34
py35
pypy
py27-pexpect
py27-xdist
py27-trial
py35-pexpect
py35-xdist
py35-trial
py27-nobyte
doctesting
freeze
docs
```